tests/conftest.py

```diff
@@ -1187,6 +1202,8 @@ def finalizer():
             config.RUN['load_status'] = 'finished'
             if thread:
                 thread.join()
+            if cluster_load_error:
+                raise cluster_load_error
```

Comment:
raise would still cause the tests to fail?

Reply:
This will fail at the end of the session as it's in the finalizer, and yep, we should see it as teardown failure I think but all tests should have been already finished.  So that case we will notice that there was an issue with this cluster load, otherwise we could miss that problem.

Reply:
this will cause the last test case in the execution to fail which is kind of false negative for this case since the real failure is in the cluster load

Reply:
Anyway we need to report this error and I cannot come with any other solution how make it visible in report.. I rather see the last test case fail with this and people who will did analysis will say that it failed cause of issue with cluster load, then we will see there was some issue with cluster load and we didn't maybe run with the load.   How else you would like to notice that error @ebenahar if we will ignore it at all?

Reply:
How about ending the complete execution since it is going to cause bunch of failures?

Reply:
Why it should cause bunch of failures?  Worst case it will not run the IO in the background if some problem with ClusterLoad fixture.   So when we spent the time on deploying the cluster and we triggered tests it's better to have results without IO in the background instead of fail whole execution and destroy the cluster.

Reply:
because you are raising an exception in the finalizer, if we truly dont want the test to fail then we shoudn't even care about the exception at the end of the thread.join()

Reply:
But then we will never notice that we had some issue with the execution and clusterLoad fixture.  As I discussed with Elad, we really want to see if there was some issue with that and hence one error which will be easily saw in the report and from the analysis they can easily tell what was the cause of that 1 test to fail.

Reply:
yes, in addition to that, the chances we will have this exception raised are lower since Petr also added retries to get_query() method of ClusterLoad in case of ScannerError exception, which adds one more protection layer